assets/js/templates/keyboard-shortcuts-help-modal.handlebars

```diff
@@ -0,0 +1,25 @@
+<div id="keyboard-shortcuts-modal-wrapper">
+  <div id="keyboard-shortcuts-modal" tabindex="-1">
+    <div class="modal-contents">
+      <div class="modal-header">
+        <div class="modal-title">Keyboard Shortcuts</div>
+        <div class="modal-close">×</div>
+      </div>
+      <div class="modal-body">
+        {{#each shortcuts}}
+          <dl class="shortcut-row">
+            <dt class="shortcut-keys">
+              {{#if this.requiresShiftKey}}
+                <kbd class="shortcut-key">shift</kbd>
```

Comment:
I don't know anything about CSS and HTML best practices but I do have a quick question: do we need the class now that we are using a very specific `kbd` tag? Would it be better to style the tag instead?

Reply:
👍 Added the styles directly to the KBd